libs/api/domains/criminal-record/src/lib/graphql/models/criminalRecord.model.ts

```diff
@@ -0,0 +1,7 @@
+import { Field, ObjectType, ID } from '@nestjs/graphql'
+
+@ObjectType()
+export class CriminalRecord {
+  @Field()
+  contentBase64!: string
```

Comment:
Wondering about this approach.
This GraphQL field is not used so it doesn't really matter (might be worth removing it). But do we plan on being able to download a Criminal Record file, like a PDF?

Another possible solution instead of returning base64 encoded file (which has a huge payload), is to use the download service. Might be worth checking out if you plan on being able to download the file via browser?

Reply:
Ah forgot to remove this model class, after I removed an unecessary function (getCriminalRecord) in the graphQL layer.

But we are going to allow users to download the criminal record as PDF (through libs/client, not libs/api - if that matters, so it is not going through the graphQL layer).
I hadn't realized we had a download service, but we also need to display the PDF in the browser, so it will look something like this:
![image](https://user-images.githubusercontent.com/6495412/145041873-6cf992ea-c4e0-4ae1-9e07-1f399767db2e.png)

Is it possible to use this download service to access the base64 string for the PDF to display in the browser (as well as being able to download the file)?

Reply:
Alright, nice!

Yeah I believe you can use it.. it's just a REST API. Implementation can be seen in `apps/download-service/*`, although you probably won't need it since if your interaction is through a REST API you can just stream it yourself :).